src/magick-image.ts

```diff
@@ -152,6 +154,35 @@ export interface IMagickImage extends IDisposable {
     compositeGravity(image: IMagickImage, gravity: Gravity, compose: CompositeOperator, point: Point, channels: Channels): void;
     compositeGravity(image: IMagickImage, gravity: Gravity, compose: CompositeOperator, point: Point, args: string): void;
     compositeGravity(image: IMagickImage, gravity: Gravity, compose: CompositeOperator, point: Point, args: string, channels: Channels): void;
+    /**
+     * Determines the connected-components of the image.
+     * 
+     * @param connectivity The number of neighbors to visit (4 or 8).
+     * @see {@link https://imagemagick.org/script/connected-components.php}
+     * 
+     * @example
```

Comment:
Not sure if I want to keep this here because that would also require me to maintain it. A description of the method and parameters and the `@see` should be sufficient?

Reply:
> Not sure if I want to keep this here because that would also require me to maintain it.

Yeah that's totally fair. Maintenance concerns are always a good reason to be wary of too much "extra" stuff.

I think `@example` theoretically shouldn't be much of a maintenance concern though, at least in this case: 

- the examples aren't very complex (just showing basic usage and return type)
- the `IMagickImage` api likely won't change drastically 

> A description of the method and parameters and the @see should be sufficient?

I feel like it's a good first-touch bit of context for how to use this in a js-specific setting. Descriptions tell you what it does, `@see` gives extra information like the various defines available, but `@example` shows _how_ to use them in js specifically.

Just sharing my thoughts though! If it still feels like something you'd prefer to not have to think about, will gladly remove

Reply:
I will remove them myself after merging the PR. I think we should put the examples somewhere else.